tests/test_utils/mock_executor.py

```diff
@@ -36,10 +36,17 @@ def __init__(self, do_update=True, *args, **kwargs):
         self.history = []
         # All the tasks, in a stable sort order
         self.sorted_tasks = []
-        self.mock_task_results = defaultdict(lambda: State.SUCCESS)
+
+        # If multiprocessing runs in spawn mode,
+        # arguments are to be pickled but lambda is not picclable.
+        # So we should pass self.success instead of lambda.
+        self.mock_task_results = defaultdict(self.success)
 
         super().__init__(*args, **kwargs)
 
+    def success(self):
+        return State.SUCCESS
```

Comment:
Where is this used?

Reply:
[Here](https://github.com/apache/airflow/pull/7128/files#diff-d7e6d11bfaeca087149ad577dd7704daR43)